src/main/scala/viper/server/vsi/Envelope.scala

```diff
@@ -0,0 +1,31 @@
+package viper.server.vsi
+
+import viper.silver.reporter.Message
+
+trait Envelope{
+}
+
+trait Letter {
+  type A
+  var message: A = _
```

Comment:
Does it make sense for ```message``` to be mutable? Maybe it could be a ```val``` parameter of the constructor of ```Letter```.